packages/client/tailwind.preset.d.ts

```diff
@@ -0,0 +1,30 @@
+type ThemeColors =
```

Comment:
I'm not sure if that's the best way, but since we want to import the file into tailwind.config.js we need to have colors also in the `.js` file. This, file adds automatic types when we import it into our typescript codebase. I didn't go with `json` as I wanted to use Tailwind presets functionality, in case we'd like a single source of truth for all our products.